components/typography/textUtils.html

```diff
@@ -6,44 +6,119 @@
+        <a href="../../index.html" id="logo">
+            <h1 class="primary-color-text fw-900">
+                Intact <span class="secondary-color-text fw-600">UI</span>
+            </h1>
+        </a>
+        <a href="../docs.html">
+            <p id="docs" class="text-s secondary-color-text fw-600">Docs</p>
+        </a>
+    </nav>
+
+    <main class="main">
+        <aside class="side-bar">
+            <div><a href="../../components/docs.html">Installation</a></div>
+            <div><a href="../typography/textUtils.html">Typography</a></div>
+            <h3>Components</h3>
+            <div><a href="../alert/alert.html">Alert</a></div>
+            <div><a href="../avatar/avatar.html">Avatar</a></div>
+            <div><a href="../badge/badge.html">Badge</a></div>
+            <div><a href="../button/button.html">Button</a></div>
+            <div><a href="../card/card.html">Card</a></div>
+            <div><a href="../grid/grid.html">Grid</a></div>
+            <div><a href="../image/image.html">Image</a></div>
+            <div><a href="../input/input.html">Input</a></div>
+            <div><a href="../toast/toast.html">Toast</a></div>
+
+        </aside>
+        <div class="content">
+            <h1>Text utilities</h1>
+            <p>
+                Text utilites include headings, different font weights, font sizes,colored texts.
+            </p>
+
+            <h3>Headings</h3>
+            <div class="demo-container vertical-container">
+                <h1>Heading 1</h1>
+                <h2>Heading 2</h2>
+                <h3>Heading 3</h3>
+                <h4>Heading 4</h4>
+                <h5>Heading 5</h5>
+                <h6>Heading 6</h6>
+            </div>
+            <div class="iframe-container">
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253EHeading%25201%253C%252Fh1%253E%2520%2520%2520%2520%2520%2520%2520%250A%253Ch2%253EHeading%25202%253C%252Fh2%253E%250A%253Ch3%253EHeading%25203%253C%252Fh3%253E%250A%253Ch4%253EHeading%25204%253C%252Fh4%253E%250A%253Ch5%253EHeading%25205%253C%252Fh5%253E%250A%253Ch6%253EHeading%25206%253C%252Fh6%253E"
+                    style="width: 1024px; height: 473px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+              </iframe>
+            </div>
+
+            <h3>Text alignments</h3>
+            <p class="text-s">To align text to left of container, use <span class="secondary-color-text fw-600">text-left</span></p>
+            <p class="text-s">To align text to center of container, use <span class="secondary-color-text fw-600">text-center</span></p>
+            <p class="text-s">To align text to right of container, use <span class="secondary-color-text fw-600">text-right</span></p>
```

Comment:
You can also use code tag to specify the class names. It will let the user know that the text specified is a code.